### PR TITLE
Neculaclaudiu/tema7

### PR DESCRIPTION
**Pull request by Claudiu357**

### CODE REVIEW

File: src/test/java/neculaclaudiu/tests/testNg/BaseTest.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+
+public class BaseTest {
+    protected ChromeDriver driver;
+    @BeforeMethod
+    public void initChromeBrowser(){
+        driver = ChromeDriverManager.createChromeDriver();
+    }
+    @AfterMethod
```
Comment (95teo) [inline src/test/java/neculaclaudiu/tests/testNg/BaseTest.java:14]: Please add a blank line between methods. Thanks!

File: src/test/java/neculaclaudiu/tests/testNg/CuraHealthcareTesttt.java [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+public class CuraHealthcareTesttt extends BaseTest{
+
+    @DataProvider(name = "loginDataProvider")
+    public Object[][] loginDataProvider(){
+        return new Object[][] {
+                {"John Doe", "ThisIsNotAPassword", "validLogin"},
+                {"Claudiu", "ThisIsAPassword", "invalidLogin"}
+        };
+    }
+    @DataProvider(name = "scenario")
```
Comment (95teo) [inline src/test/java/neculaclaudiu/tests/testNg/CuraHealthcareTesttt.java:19]: Please add a blank line between methods. Thanks!

File: src/test/resources/neculaclaudiu/suite-testng.xml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd" >
+
+<suite name="SuitaTestNG" verbose="1" >
+    <test name="AltTest1" >
+        <classes>
+            <class name="neculaclaudiu.teorie.altPachet.AltTest" />
+        </classes>
+    </test>
+
+    <test name="DemoTest">
```
Comment (95teo) [inline src/test/resources/neculaclaudiu/suite-testng.xml:10]: You can remove lines from 10-15.

File: src/test/resources/neculaclaudiu/suite-testng.xml [hunk 4]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd" >
+
+<suite name="SuitaTestNG" verbose="1" >
+    <test name="AltTest1" >
+        <classes>
+            <class name="neculaclaudiu.teorie.altPachet.AltTest" />
```
Comment (95teo) [inline src/test/resources/neculaclaudiu/suite-testng.xml:6]: Please put the right path to CuraHealthcareTest class.

File: src/test/java/neculaclaudiu/tests/testNg/CuraHealthcareTesttt.java [hunk 5]
```diff
@@ -0,0 +1,105 @@
+package neculaclaudiu.tests.testNg;
+
+import com.beust.ah.A;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+@Test
+public class CuraHealthcareTesttt extends BaseTest{
```
Comment (95teo) [inline src/test/java/neculaclaudiu/tests/testNg/CuraHealthcareTesttt.java:10]: Please remove the extra 't's from name of the class at the end. Leave only one 't'. CuraHealthcareTesttt -> CuraHealthcareTest. Thanks!

**Review by 95teo — CHANGES_REQUESTED** [review]

**Review by 95teo — APPROVED** [review]

**Comment by 95teo** [comment]
Please don't forget to ask for review from colleagues too. After you have at least 2 approvals (1 from me required) you can merge this PR. Thanks!

**Review by sorinfoca — APPROVED** [review]

**Pull request merged**